include/triton/Dialect/Triton/IR/TritonAttrDefs.td

```diff
@@ -29,12 +29,20 @@ def TT_RedOpAttr : I32EnumAttr<
     /*case*/
     [
         I32EnumAttrCase</*sym*/"ADD", 1, /*str*/"add">,
-        I32EnumAttrCase<"MAX", 2, "max">,
+        I32EnumAttrCase<"FADD", 2, "fadd">,
```

Comment:
An off-topic question to @ptillet, since there are already explicit dtypes for both the operand and result in the definition of tt.reduce, is it still necessary to distinguish ADD/FADD (or ARGMAX/ARGUMAX) here? Same question for the definition of tt.atomic_rmw.

Reply:
it's not necessary, but I think it simplifies the semantics a little since what it says is that essentially the tag just refers to the binary operator to be applied to the accumulator during the reduction.